src/Status.js

```diff
@@ -0,0 +1,24 @@
+import LocalStorage from './localstorage.js';
+
+export default class Status {
+  static statusChanged(checkBox, index, target, tasksList) {
+    const currentLi = target.parentElement;
+    const taskDescription = currentLi.querySelector('#task');
+
+    if (checkBox.checked) {
+      taskDescription.classList.add('strike');
+      tasksList[index].completed = true;
+    } else {
+      taskDescription.classList.remove('strike');
+      tasksList[index].completed = false;
+    }
+
```

Comment:
- [ ]  Your project has one issue that you should fix, when there is many checked tasks, and the user deletes one of those checked tasks, the rest are getting unchecked. please fix this to go to the next milestone.